src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -129,7 +129,8 @@ public void recallRequestWithMGDAndRDValuesChangesDueDateToRD()
 
     useLoanPolicyAsFallback(loanPolicy.getId(),
         requestPoliciesFixture.allowAllRequestPolicy().getId(),
-        noticePoliciesFixture.inactiveNotice().getId());
+        noticePoliciesFixture.inactiveNotice().getId(),
+        overdueFinePoliciesFixture.facultyStandard().getId());
```

Comment:
Can you extract this repeated code into a method, for example

    private void setFallbackPolicies(LoanPolicyBuilder loanPolicyBuilder) {
      final IndividualResource loanPolicy = loanPoliciesFixture.create(loanPolicyBuilder);
      useLoanPolicyAsFallback(loanPolicy.getId(),
          requestPoliciesFixture.allowAllRequestPolicy().getId(),
          noticePoliciesFixture.inactiveNotice().getId(),
          overdueFinePoliciesFixture.facultyStandard().getId());
    }

Reply:
@julianladisch I don`t think that it is a really good idea. Because, if I`ll create this method with these arguments in "useLoanPolicyAsFallback" method, then there needs to be have other "setFallbackPolicies" method, with, for example, "activeNotice" instead of  ''inactiveNotice" (such as in your method).  The same for allowAllRequestPolicy and facultyStandard (there can be other options). So, in this approach, we need to have a specific method for a specific case.  What do you think about it? Does it make sense?

Reply:
This has been fixed, thank you!